server/auth_token_test.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"encoding/json"
+	"errors"
+	"io"
+	"testing"
+
+	"bou.ke/monkey"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/oauth2"
+)
+
+type mockBLock struct{}
+
+func (b *mockBLock) BlockSize() int { return 0 }
+
+func (b *mockBLock) Encrypt(_, _ []byte) {}
+
+func (b *mockBLock) Decrypt(_, _ []byte) {}
+
+type mockAesgcm struct{}
+
+func (a *mockAesgcm) NonceSize() int { return 1 }
```

Comment:
```suggestion
func (a *mockAesgcm) NonceSize() int { return 0 }
```
The default value of int is 0. Change in the next func also

Reply:
test case fails on changing this value to 0